datafusion/core/tests/sqllogictests/test_files/ddl.slt

```diff
@@ -28,6 +28,18 @@ select * from users;
 1 2
 2 3
 
+statement ok
+drop table users
+
+statement ok
```

Comment:
Can you also please add a negative test (aka that calling `select * INTO users ...` will error if `users` already exists?